test/uk/gov/hmrc/gform/sharedmodel/JsonParseTestFormat.scala

```diff
@@ -181,4 +181,52 @@ class JsonParseTestFormat extends Spec with TableDrivenPropertyChecks {
     }
   }
 
+  "A text area component with a valid no of rows" should "parse correctly" in {
+
+    val multilineCombinations = Table(
```

Comment:
dont think we need Table driven tests here. You are not testing any variation. A simple example with 1 or 11 rows is good enough. Avoid magic numbers like 7, 10 , 40 etc

Reply:
Have changed to a single value test. But there is no harm to test with multiple values.

Reply:
No, but while reading it can get confusing when such magic numbers are seen. Does it only work for these 3 numbers. As far as possible, stick to simple values like 1, A etc